packages/datadog-plugin-elasticsearch/src/index.js

```diff
@@ -24,8 +24,8 @@ function createWrapRequest (tracer, config) {
         }
       })
 
-      if (params.body) {
-        span.setTag('elasticsearch.body', JSON.stringify(params.body))
+      if (params.body || params.bulkBody) {
```

Comment:
There should be a corresponding test for this. Also, is it possible that both could be set or it's always one or the other?